### PR TITLE
New DKP graphic

### PR DESCRIPTION
**Pull request by Colin1841**
No Jira, just some cleanup to replace the old graphic with the new one.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4532.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Review by cmastr — APPROVED** [review]

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kommander/2.2/introduction/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 As the leading independent platform for Kubernetes in production, the D2iQ Kubernetes Platform (DKP) provides a holistic approach and a complete set of enterprise-grade technologies, services, training, and support to build and run applications in production at scale. Built around the open-source Cluster API, the new version of DKP becomes the single, centralized point of control for an organization’s application infrastructure, empowering organizations to more easily deploy, manage, and scale Kubernetes workloads in Day 2 production environments.
 
-![image](../img/dkp_platform2.png)
+![image](../img/platform_diagram_dark.png)
```
Comment (msdolbey) [inline pages/dkp/kommander/2.2/introduction/index.md:12]:
```suggestion
![DKP Kubernetes Platform Multi Cluster Management](../img/platform_diagram_dark.png)
```

File: pages/dkp/kommander/2.3/introduction/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 As the leading independent platform for Kubernetes in production, the D2iQ Kubernetes Platform (DKP) provides a holistic approach and a complete set of enterprise-grade technologies, services, training, and support to build and run applications in production at scale. Built around the open-source Cluster API, the new version of DKP becomes the single, centralized point of control for an organization’s application infrastructure, empowering organizations to more easily deploy, manage, and scale Kubernetes workloads in Day 2 production environments.
 
-![image](../img/dkp_platform2.png)
+!![image](../img/platform_diagram_dark.png)
```
Comment (msdolbey) [inline pages/dkp/kommander/2.3/introduction/index.md:12]:
```suggestion
![DKP Kubernetes Platform Multi Cluster Management](../img/platform_diagram_dark.png)
```

File: pages/dkp/konvoy/2.2/intro-dkp/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 As the leading independent platform for Kubernetes in production, the D2iQ Kubernetes Platform (DKP) provides a holistic approach and a complete set of enterprise-grade technologies, services, training, and support to build and run applications in production at scale. Built around the open-source Cluster API, the new version of DKP becomes the single, centralized point of control for an organization’s application infrastructure, empowering organizations to more easily deploy, manage, and scale Kubernetes workloads in Day 2 production environments.
 
-![image](../img/dkp_platform2.png)
+![image](../img/platform_diagram_dark.png)
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.2/intro-dkp/index.md:12]:
```suggestion
![DKP Kubernetes Platform Multi Cluster Management](../img/platform_diagram_dark.png)
```

File: pages/dkp/konvoy/2.3/intro-dkp/index.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 As the leading independent platform for Kubernetes in production, the D2iQ Kubernetes Platform (DKP) provides a holistic approach and a complete set of enterprise-grade technologies, services, training, and support to build and run applications in production at scale. Built around the open-source Cluster API, the new version of DKP becomes the single, centralized point of control for an organization’s application infrastructure, empowering organizations to more easily deploy, manage, and scale Kubernetes workloads in Day 2 production environments.
 
-![image](../img/dkp_platform2.png)
+![image](../img/platform_diagram_dark.png)
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.3/intro-dkp/index.md:12]:
```suggestion
![DKP Kubernetes Platform Multi Cluster Management](../img/platform_diagram_dark.png)
```

**Review by msdolbey — APPROVED** [review]
Nice pics!

File: pages/dkp/konvoy/2.2/intro-dkp/index.md (hunk 3 above)
Comment (Colin1841) [reply pages/dkp/konvoy/2.2/intro-dkp/index.md:12]: Thanks @msdolbey - that'll look a little better!

**Comment by msdolbey** [comment]
linting error seems irrelevant - I say it's a go whenever you're ready!

**Pull request merged**